src/index.ts

```diff
@@ -327,6 +342,83 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     return this.sendRawBundle(signedTransactions, targetBlockNumber, opts)
   }
 
+  public async sendPrivateTransaction(
+    transaction: FlashbotsBundleTransaction,
+    opts?: {
+      maxBlockNumber?: number,
+      minTimestamp?: number,
+    },
+  ): Promise<FlashbotsPrivateTransaction> {
+    return this.sendPrivateRawTransaction(
+      await transaction.signer.signTransaction(transaction.transaction),
+      opts,
+    )
+  }
+
+  public async sendPrivateRawTransaction(
+    signedTransaction: string,
+    opts?: {
+      maxBlockNumber?: number,  // highest block # in which the bundle might be included
+      minTimestamp?: number,    // sim timestamp
+    },
+  ): Promise<FlashbotsPrivateTransaction> {
+    const params = {
+      tx: signedTransaction,
+      maxBlockNumber: opts?.maxBlockNumber,
```

Comment:
`eth_sendPrivateTransaction` does not support `maxBlockNumber`

Reply:
This is my bad, I took a wrong look at the relay code. What are the options?

Reply:
Sorry, my bad actually, we do actually support `maxBlockNumber` 🤦